docs/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/org-service-migration.md

```diff
@@ -76,8 +75,9 @@ Make sure that the following configurations are not included in your Agent run c
 
 | Environment Variable                                       | Default | Description |
 |------------------------------------------------------------|---------|-------------|
-| `MODERNE_AGENT_ORGANIZATION_REPOSCSV`                      |         | The path of your `repos.csv` file that provides organization information. |
-| `MODERNE_AGENT_ORGANIZATION_DEVCENTER`                     |         | The path of your `devcenter.json` file that provides the DevCenter configurations. |
+| `MODERNE_AGENT_ORGANIZATION_REPOSCSV`                      |         | The path of your `repos.csv` file that provides organization information. This could also be an unauthenticated HTTP/S URI in the form of `https://your-serve/repos.csv`. |
+| `MODERNE_AGENT_ORGANIZATION_DEVCENTER`                     |         | The path of your `devcenter.json` file that provides the DevCenter configurations. This could also be an unauthenticated HTTP/S URI in the form of `https://your-service/devcenter.json`. |
```

Comment:
You have `your-serve` on the first line and `your-service` on the second, I assume you're trying to show that they can be different services, but maybe then you can do something like `https://<internal-endpoint>`